### DISCUSSION TITLE
Error with Konsta after following Framework7 Vue integration

### DISCUSSION THREAD

**Discussion by Daniel-Mendes** [konstaui/konsta#193] [Q&A]
# Reproduction

TLDR: Repo to reproduce the error https://github.com/Daniel-Mendes/framework7-vue-konsta

1. Install Framework7 template with vue and capacitor

`$ framework7 create --ui`

2. Changing `postcss.config.js` , `tailwind.config.js`.

3. Adding `k-provider` to App.vue

```vue
<template>
  <k-provider theme="parent">
    <f7-app v-bind="f7params">
    
    ...
```

4. Adding a test component in Home.vue

```vue
<script>
import { kButton } from 'konsta/vue'

export default {
  components: {
    kButton
  }
}
</script>

<template>
...
 <k-button>Test</k-button>
...
 </template>
```

Getting error in console:

`
runtime-core.esm-bundler.js:39 [Vue warn]: inject() can only be used inside setup() or functional components.
warn$1 @ runtime-core.esm-bundler.js:39
`

`
runtime-core.esm-bundler.js:39 [Vue warn]: Unhandled error during execution of scheduler flush. This is likely a Vue internals bug. Please open an issue at https://github.com/vuejs/core .
warn$1 @ runtime-core.esm-bundler.js:39
`

`
use-theme-classes.js:48 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'value')
    at use-theme-classes.js:48:25
    at runtime-core.esm-bundler.js:7892:14
    at ReactiveEffect.fn (reactivity.esm-bundler.js:982:13)
    at ReactiveEffect.run (reactivity.esm-bundler.js:170:19)
    at get value [as value] (reactivity.esm-bundler.js:993:61)
    at triggerComputed (reactivity.esm-bundler.js:189:19)
    at get dirty [as dirty] (reactivity.esm-bundler.js:144:11)
    at get value [as value] (reactivity.esm-bundler.js:992:41)
    at triggerComputed (reactivity.esm-bundler.js:189:19)
    at get dirty [as dirty] (reactivity.esm-bundler.js:144:11)
`

**Comment by akhilnarayanan1** [comment]
any solution to this as of now?
